freqtrade/freqai/freqai_interface.py

```diff
@@ -674,7 +675,7 @@ def train(self, unfiltered_dataframe: DataFrame, pair: str, dk: FreqaiDataKitche
         """
 
     @abstractmethod
-    def fit(self, data_dictionary: Dict[str, Any]) -> Any:
+    def fit(self, data_dictionary: Dict[str, Any], dk: FreqaiDataKitchen) -> Any:
```

Comment:
do we assume we'll have the need to extend this interface further?
if yes - i think we should consider recommending people add either `*args, **kwargs` - or just `**kwargs` to theyr implementation (as we do with strategy methods) - so we can extend the interface at will.

Please note: 
the latter approach (`**kwargs` only) means we MUST call all additional arguments as keyword arguments (no longer as positional arguments).

Reply:
sounds good